scripts/pyenv_setup.sh

```diff
@@ -71,23 +76,25 @@ install_pyenv() {
     echo "Installing Python (if missing) via pyenv"
     local pyenv_version
     pyenv_version=$(pyenv -v | awk '{print $2}')
-    python_version=$(xargs -n1 < .python-version)
+    python_version=$(xargs -n1 <.python-version)
+    # NOTE: Older pyenv does not have access to the latest Python we require
+    if [[ "$pyenv_version" < 1.2.26 ]]; then
+      echo >&2 "!!! Your pyenv is old and does not know how to find the Python we require. "
+      echo >&2 "Run \`brew update && brew upgrade pyenv\` (this is slow) and try again."
```

Comment:
I'd prefer (in general) that when we suggest to run commands that they should be on their own line so that 1) it stands out and 2) it is very easy to copy and paste.

```
!!! Your pyenv is old and does not know how to find the Python we require. Run the following and try again (this will be slow)

    brew update && brew upgrade pyenv

```